### PR TITLE
[F-039] 튜토리얼 레벨 조작키 설명 추가

### PR DESCRIPTION
**Pull request by ahchowww**
# [F-039] 튜토리얼 레벨 조작키 설명 추가
## PR Description
- WASD+R key usage and time rewind key explanations that disappear after three inputs
- Add explanation for box and lever mechanics: instruction will occur only if the player is within a 3x3 radius of the object
### Changes Included
- [x] Added new feature(s)
- [ ] Fixed identified bug(s)
- [ ] Updated relevant documentation
### Notes for Reviewer
---
## Reviewer Checklist
- [x] Code is written in clean, maintainable, and idiomatic form.
- [x] Automated test coverage is adequate.
- [x] All existing tests pass.
- [x] Manual testing has been performed to ensure the PR works as expected.
- [x] Code review comments have been addressed or clarified.
---
## Additional Comments

linter.yml:

### CODE REVIEW

**Review by parkchaehyun — COMMENTED** [review]
우선 develop에 rebase했습니다

File: Chronus/Assets/Scripts/TutorialManager.cs [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+public class TutorialManager : MonoBehaviour
+{
+    public GameObject gameInstructionCanvas;
+    public GameObject leverInstructionCanvas;
+    public GameObject timeRewindInstructionCanvas;
+    public GameObject boxInstructionCanvas;
+
+    public Transform player;
+    public Transform lever;
+    public Transform box;
```
Comment (parkchaehyun) [inline Chronus/Assets/Scripts/TutorialManager.cs:15]: 어차피 같은 transform type이니 public Transform object 정도로 하나만 사용해서 tile, lever, box 모두 사용하면 될 것 같습니다

setlever, setbox, clearlever, clearbox도 set clear 하나씩으로 합치구요

**Review by parkchaehyun — CHANGES_REQUESTED** [review]

File: Chronus/Assets/Scripts/TutorialManager.cs (hunk 1 above)
Comment (ahchowww) [reply Chronus/Assets/Scripts/TutorialManager.cs]: 반영했습니다.

Screenshot 2024-12-16 004311

GameInstructionCanvas와 TimeRewindInstructionCanvas에도 동일한 방법을 적용했습니다. StartTile 근처에 있는 타일에 'TutorialTile' 태그를 추가했습니다

**Review by parkchaehyun — APPROVED** [review]
변경해주신 것 확인했습니다. 고생하셨습니다.

**Pull request merged**